### ISSUE TITLE
Deployment error

### ISSUE THREAD

**Issue by RogerSchmidlin** [davidortinau/WeatherTwentyOne#24]
I cloned the project and ran it with the newest everything 16.11.0. I get the following deployment error. Any ideas?:

DEP1700: The recipe file "C:\Users\xhor606\source\repos\MauiWeatherApp\src\WeatherTwentyOne.WinUI3 (Package)\bin\x86\Debug\WeatherTwentyOne.WinUI3 (Package).build.appxrecipe" does not exist. You may need to build your project.	WeatherTwentyOne.WinUI3 (Package)

**Comment by TitusRie** [comment]
Looks like a rebuild/redeploy just fixes this

**Comment by chrkon** [comment]
No, a rebuild/redeploy will not fix this. I have the same error on my system.
Visual Studio 2022 Community Edition, german
Rebuild works, but deploy stops with the message "xxx.build.appxrecipe does not exist."
The same project on a second PC with Visual Studio 2022 prof., english version is working and I get no error message on deploying.
